src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+function addSpan([...elements]) {
```

Comment:
It's better to use a default parameter for `elements` instead of spreading the arguments. Example: `function addSpan(elements = []) {...}`